src/uploaders/lib/FormatErrorLog.ts

```diff
@@ -14,7 +14,8 @@ function formatErrorLog (e: NetworkError): string {
       str += `\n\n  responseText = ${e.responseText}`
       break
     case NetworkErrorCode.DUPLICATE:
-      str += 'A source map matching the same criteria has already been uploaded. If you want to replace it, use the "overwrite" flag.'
+      str += 'A source map matching the same criteria has already been uploaded. ' +
+        'If you want to replace it, use the "overwrite" flag (or remove the "no-overwrite" flag).'
```

Comment:
I think we should special case this message for React Native as the `overwrite` flag doesn't exist anymore (and `no-overwrite` doesn't exist in browser/node) so it's a bit confusing like this

The easiest ways I can think of is to either check the error code before calling this method in the ReactNativeUploader or pass an `isReactNative` boolean (default `false` so no where else needs to change)